package/test/test.ts

```diff
@@ -0,0 +1,5 @@
+import _VAPI from "../src/index.js"
+const VAPI = new _VAPI()
+
+const res = await VAPI.getMMR("jameslinimk", "8868", "na")
+console.log(res)
```

Comment:
Should be removed or replaced by real tests

Reply:
I was just using it to make sure the function works, I'll write real tests soon